src/config.rs

```diff
@@ -831,6 +842,35 @@ impl TiKvConfig {
             }
             self.raft_store.region_split_size = default_raft_store.region_split_size;
         }
+        if self.server.end_point_concurrency != None {
```

Comment:
Concurrency config is not applicable to the old style, since it is calculated from current CPU cores. When user specifies a concurrency config, no matter it is equal to the calculated value or not, user configuration should be respected. Due to these reasons, I uses `None` to indicate that if user has specified a concurrency (!None) then it is used as current config.